src/Apps/Artist/Components/ArtistHeader.tsx

```diff
@@ -102,7 +102,7 @@ export class LargeArtistHeader extends Component<Props> {
     return (
       <Box width="100%">
         {hasImages && (
-          <Fragment>
+          <section>
```

Comment:
Ahh yeah, i've seen weirdness with fragments in the past (just in general, not involving SSR)